htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementDecorator.java

```diff
@@ -50,26 +50,25 @@ public HtmlElementDecorator(CustomElementLocatorFactory factory) {
     }
 
     public Object decorate(ClassLoader loader, Field field) {
-        if (isTypifiedElement(field)) {
-            return decorateTypifiedElement(loader, field);
+        try {
```

Comment:
so, can you only add `try-catch` without changing the `if` style to `elif`? Previous variant is more readable for me